std_msgs/Header header 
uint16 chan1
uint16 chan2
uint16 chan3
uint16 chan4
uint16 chan5
uint16 chan6
uint16 chan7
uint16 chan8
uint16 chan9
uint16 chan10
uint16 chan11
uint16 chan12
uint16 chan13
uint16 chan14
uint16 chan15
uint16 chan16
